google/cloud/spanner/internal/database_admin_metadata_test.cc

```diff
@@ -95,12 +96,12 @@ TEST_F(DatabaseAdminMetadataTest, UpdateDatabase) {
   DatabaseAdminMetadata stub(mock_);
   CompletionQueue cq;
   gsad::v1::UpdateDatabaseDdlRequest request;
-  request.set_database(
+  request.set_database(internal::UrlEncode(
```

Comment:
Changing the contents of the request is cheating.

You might as well update the library code: https://github.com/googleapis/google-cloud-cpp/blob/dc3523ebe911bdcb492ce8dbe6de1ddd22c75b57/google/cloud/spanner/internal/database_admin_metadata.cc#L36

Reply:
Fixed.